src/github.com/matrix-org/dendrite/clientapi/readers/profile.go

```diff
@@ -38,6 +43,12 @@ type displayName struct {
 	DisplayName string `json:"displayname"`
 }
 
+type profileUpdate struct {
+	Updated  string `json:"updated"`
```

Comment:
Could you add some comments to explain what "updated", "old_value", and "new_value" mean please?